src/mode/modeNormal.ts

```diff
@@ -13,20 +14,30 @@ export default class CommandMode extends Mode {
 		this.keyHandler = {
 			":" : () => { showCmdLine(); },
 			"u" : () => { vscode.commands.executeCommand("undo"); },
-			"ctrl+r" : () => { vscode.commands.executeCommand("redo"); },
+			"ctrl\+r" : () => { vscode.commands.executeCommand("redo"); },

```

Comment:
Ugh sorry that was actually a left over, when I was experimenting with how to hand fx and Fx I at one point turned the whole table into an array of regex's instead.  The thought at the the time was then f. and F. could be matched, but I didn't like that you for each handle you would have to loop through the array of matches, rather than like now where it is a direct lookup via the associative array.